carla_integration/docker-compose-background.yml

```diff
@@ -17,7 +17,7 @@ version: '2'
 
 services:
   web-ui:
-    image: usdotfhwastol/carma-web-ui:carma-system-4.1.0
+    image: usdotfhwastol/carma-web-ui:carma-system-4.2.0
```

Comment:
Let's revert this back to 3.9 as well, since this also is for XIL and thus not supporting ROS2. Must have missed this in a previous release, seeing as it's 4.1 previously.

Reply:
updated